### YamlMime:ManagedReference
items:
- uid: System.IO.IsolatedStorage.IsolatedStorageException
  id: IsolatedStorageException
  children:
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException
  fullName: System.IO.IsolatedStorage.IsolatedStorageException
  type: Class
  summary: "Исключение, возникающее при сбое операции в изолированном хранилище."
  remarks: "`IsolatedStorageException`использует ISS_E_ISOSTORE HRESULT, который имеет значение 0x80131450.       Ниже приведены наиболее распространенные примеры исключений изолированного хранения.      — Свидетельство отсутствует. Изолированного хранения необходимо свидетельство (сведения о сборке и ее источнике) чтобы определить удостоверение кода и подключите его к пространство правильное связанного файла. Без этой информации нельзя использовать изолированное хранилище.      -Недопустимые операции. Некоторые <xref:System.IO.FileStream>операции не поддерживаются для изолированного хранилища.</xref:System.IO.FileStream>       Список начальных значений свойств для экземпляра `IsolatedStorageException`, в разделе <xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>конструкторов.</xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class IsolatedStorageException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  id: '#ctor'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> класса со свойствами по умолчанию."
  remarks: "В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка локализованного сообщения об. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException ();
    parameters: []
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> класса с указанным сообщением об ошибке."
  remarks: "В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> сериализованными данными."
  remarks: "Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке."
  syntax:
    content: protected IsolatedStorageException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Объект, содержащий сериализованные данные объекта."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Контекстные сведения об источнике или назначении."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;> </xref> класса с указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение."
  remarks: "Исключение, созданное как прямой результат предыдущего исключения, должно содержать ссылку на предыдущее исключение в <xref:System.Exception.InnerException%2A>свойство.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Свойство возвращает то же значение, которое передается конструктору, или `null` Если <xref:System.Exception.InnerException%2A>свойства не предоставляет конструктору значение внутреннего исключения.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Ссылка на внутреннее исключение. |   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
    - id: inner
      type: System.Exception
      description: "Исключение, которое стало причиной текущего исключения. Если `inner` не <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, текущее исключение сгенерировано в <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> блок, обрабатывающем внутреннее исключение."
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException.IsolatedStorageException
